src/java.desktop/share/classes/javax/swing/plaf/synth/SynthLookAndFeel.java

```diff
@@ -615,6 +615,16 @@ public void load(InputStream input, Class<?> resourceBase) throws
      * <code>new URL(synthFile, path)</code>. Refer to
      * <a href="doc-files/synthFileFormat.html">Synth File Format</a> for more
      * information.
+     * <p>
+     * Whilst this API may be safe for loading local resources that are
+     * delivered with a {@code LookAndFeel} or application, and so have an
+     * equal level of trust with application code, using it to load from
+     * from remote resources, particularly any which may have a lower level of
+     * trust, is strongly discouraged.
+     * The alternative mechanisms to load styles from an {@code InputStream}
+     * using a resource co-located with the application or by
+     * providing a {@code SynthStyleFactory} are preferred.
+     * Consequently this method is deprecated and will be removed in a future release.
```

Comment:
We probably can use "@see" tag to link the alternate preferred method `load(InputStream input, Class<?> resourceBase)` since this is going to be deprecated

Reply:
Great suggestion. The javadoc should link to the alternatives `load` or `SynthStyleFactory`.

The following line should be added to javadoc:
```java
     * @deprecated Use {@link #load(InputStream, Class)} or {@link SynthStyleFactory}
     *             as described above.
```
The entire paragraph could probably used in `@deprecated`. The result would look like this:
![Rendered javadoc with @deprecated before the added paragraph](https://user-images.githubusercontent.com/70774172/215103931-2f093a8a-f057-49f7-8dce-e8cbb9e2ff90.png)

Here's the code that I used:
```java
     * information.
     *
     * @deprecated Whilst this API may be safe for loading local resources that are
     * ...
     * The alternative mechanisms to load styles from an
     * {@link #load(InputStream, Class) InputStream} using a resource
     * co-located with the application or by
     * providing a {@link SynthStyleFactory} are preferred.
     * ...
```
However, such use of linking for `load` with `InputStream` is likely to be confusing, so the text should be re-worded.